internal/cli/kraft/net/create/create.go

```diff
@@ -56,9 +63,13 @@ func NewCmd() *cobra.Command {
 	return cmd
 }
 
-func (opts *CreateOptions) Pre(cmd *cobra.Command, _ []string) error {
+func (opts *CreateOptions) Pre(cmd *cobra.Command, args []string) error {
 	opts.Driver = cmd.Flag("driver").Value.String()
 
+	if len(args[0]) > defaultNetworkNameMaxSize {
```

Comment:
Thanks @craciunoiuc!
One last thing, can you please move this to ``Run`` and also add a check ``len(args) > 0``? Some places including compose use this as an API directly :pray:

Reply:
sure thingy